x-pack/plugins/observability_solution/inventory/public/components/search_bar/discover_button.tsx

```diff
@@ -38,14 +35,10 @@ export function DiscoverButton({ dataView }: { dataView: DataView }) {
 
   const filters: PhrasesFilter[] = [];
 
-  const entityTypeField = dataView.getFieldByName(ENTITY_TYPE);
-
-  if (entityTypes && entityTypeField) {
-    const entityTypeFilter = buildPhrasesFilter(entityTypeField, entityTypes, dataView);
-    filters.push(entityTypeFilter);
```

Comment:
We are currently passing an empty array with the filters after removing this - we should pass the filters from the unified search + controls. We can do that after the [PR](https://github.com/elastic/kibana/pull/199306) is merged as it will be moved to a new hook and it will make it easier to get them from the state there and use it for all the links